homeassistant/components/pooldose/strings.json

```
@@ -0,0 +1,149 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Set up SEKO PoolDose device",
+        "description": "The device password must be set to default (0000).",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        },
+        "data_description": {
+          "host": "IP address or hostname of your device"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "api_not_set": "API version not found in device response",
+      "api_not_supported": "Unsupported API version {api_version_is} (shall: {api_version_should})",
+      "params_fetch_failed": "Unable to fetch core parameters from device",
+      "no_device_info": "Unable to retrieve device information. The device may not be properly initialized or may be an unsupported model.",
+      "no_serial_number": "No serial number found on the device. The device may not be properly configured or may be an unsupported model.",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "orp": {
+        "name": "ORP"
+      },
+      "ph_type_dosing": {
+        "name": "pH dosing type",
+        "state": {
+          "alcalyne": "pH+",
+          "acid": "pH-"
+        }
+      },
+      "peristaltic_ph_dosing": {
+        "name": "ph peristaltic dosing",
```

"pH" needs the correct spelling here, too:

```suggestion
        "name": "pH peristaltic dosing",
```